### DISCUSSION TITLE
Visual Studio Code Duplication Detection not off

### DISCUSSION THREAD

**Discussion by huuse** [community/community#76510] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

Hi!

What am I missing, in github.com i have 
- "Suggestions matching public code" = "Allow"

But in Visual Studio Code i still get: "Sorry, the response matched public code so it was blocked. Please rephrase your prompt".

Please help

**Comment by Akash1134** [comment, marked as answer]
Hi @huuse ,

We're sorry about the trouble caused. It seems you might be impacted by a bad deploy that happened [yesterday](https://github.com/microsoft/vscode-copilot-release/issues/583) 

Please restart VSCode and logout/login to the Copilot extension. This issue has already been resolved.

Thanks!

**Reply by huuse** [reply]
Thank you. That solved the issue!
